include/exadg/structure/driver.h

```diff
@@ -97,9 +97,6 @@ class Driver
 
   std::shared_ptr<MultigridMappings<dim, Number>> multigrid_mappings;
 
-  // user parameters
```

Comment:
why was this actually in there? just something someone forgot to delete? Probably ownership of parameters went from the driver to the application?

Reply:
@nfehn did you see this?